sqlgen/db.go

```diff
@@ -55,17 +53,26 @@ func NewDB(conn *sql.DB, schema *Schema) *DB {
 				return nil, err
 			}
 			clause, args = selectQuery.ToSQL()
+			span.SetTag("query", clause)
+			span.SetTag("batchItemCount", len(items))
+			if marshaledItems, err := json.Marshal(filters); err == nil {
+				span.SetTag("batchItems", string(marshaledItems))
+			}
+			span.SetTag("queryVariables", args)
```

Comment:
According to the opentracing godocs: "Tag values can be numeric types, strings, or bools. "

Rather than relying on undefined behavior, we should log the string: fmt.Sprintf("%v", args)